test/mapmatch.cc

```diff
@@ -1428,7 +1428,7 @@ TEST(Mapmatch, openlr_parameter_true_osrm_api) {
       "CwOduyULYiKJAAAV//0iGw==",
       "CwOdxCULYCKJAAAN//4iGw==",
       "CwOdySULXyKJAAAf//EiGw==",
-      "CwOd1yULVyKLAABV/9AiGw==",
+      "CwOd1yULVyKLAQBV/9AiGw==",
```

Comment:
Was this change intended for this PR?

Reply:
I just updated the description/name of the PR. It actually fixes a bug in the openlr serialization at https://github.com/valhalla/valhalla/pull/2688/files#diff-dd901075ccbb89ba7900fb2348eac4f9da77488c897257ba9f657a10603531a8R80

The rename (most of the diff) is only made to avoid a similar issue in future.

Reply:
So yes, this change in test-value is expected.

Reply:
Yeah I had the same confusion. I missed the line you pointed above, thinking the diff was simply a `s/length/length_km` 🙂

Reply:
👍